reference/OpenAPI.json

```diff
@@ -41193,116 +41201,102 @@
         "type": "object",
         "title": "Metadata Definitions",
         "description": "Object representing metadata definitions.",
-        "properties": {
+        "additionalProperties": {
+          "type": "object",
+          "title": "Metadata Definitions",
+          "description": "Custom definition name. This is also shown in **Project Settings** > **Metadata Schema** in the Voucherify Dashboard.",
           "properties": {
```

Comment:
Musiałbym to sprawdzić jeszcze na readme, ale jak patrzę na kod, to chyba nam to nie zbuduje tego obiektu z response:

```
{
    "id": "ms_65DinNsL7FkBifqw2dMe6ejR",
    "related_object": "voucher1",
    "properties": {
        "my_prop": {
            "type": "string",
            "array": false,
            "optional": true,
            "object_type": null
            // w zaleznosci od type zwracane sa te property ktore mozemy podac w payload
        }
    },
    "allow_defined_only": null,
    "created_at": "2024-04-07T15:49:59.670Z",
    "updated_at": null,
    "object": "metadata_schema"
}
```

Przykład z PR-ki Piotra P.

Chodzi o to, że tu się zwraca obiekt `properties`, który ma w sobie te metadanowe properties.

Reply:
tak to prawda, już poprawiam